bundles/org.openhab.binding.urtsi/src/main/resources/OH-INF/i18n/urtsi_de.properties

```diff
@@ -10,9 +10,9 @@ urtsiDeviceCommandIntervalLabel = Befehlausführungsintervall
 urtsiDeviceCommandIntervalDescription = Die Zeit (in ms), die das Binding zwischen dem Absenden von zwei Befehlen warten soll
 
 rtsDeviceLabel = Somfy RTS Gerät
-rtsDeviceDescription = Dies ist ein Gerät, das über RTS kommuniziert (z.B. ein Rolladenmotor).
+rtsDeviceDescription =  Dies ist ein Gerät, das über RTS kommuniziert (z.B. ein Rolladenmotor).
 rtsDeviceChannelLabel = Kanal
-rtsDeviceChannelDescription = Der Kanel, der dem RTS Gerät am URTSI II zugewiesen wurde
+rtsDeviceChannelDescription =  Der Kanel, der dem RTS Gerät am URTSI II zugewiesen wurde
```

Comment:
It looks like an unexpected leading space was added.
@cweitkamp : could you please fix that.